package.json

```diff
@@ -28,11 +28,11 @@
     "type": "git",
     "url": "https://github.com/fhir/sushi.git"
   },
-  "main": "dist/app.js",
+  "main": "dist/index.js",
```

Comment:
> Because the classes most likely to be of use in other project are the FSH and FHIR types, they are added as exports to index.js. Note that the change to package.json which specifies index.js allows these classes to be included more easily, but does mean that fsh-sushi can no longer be run from its directory with a simple "node ." command.

This is just a change for developers you mean right? We can't run sushi by doing `node .` anymore, but this doesn't change anything from the perspective of a user. I'm pretty sure, but just want to double check.

Reply:
That's correct: when sushi is installed, the `bin` section of `package.json` is what defines the command-line operation. So, `sushi` will still run `dist/app.js`.

Reply:
And I don't think it's a big deal for developers anyway because I think we usually should be doing `ts-node src/app.ts` anyway...